api/XRPermissionDescriptor.json

```diff
@@ -1,200 +0,0 @@
-{
-  "api": {
-    "XRPermissionDescriptor": {
```

Comment:
This is a dictionary, which I don't think we should have in BCD but sometimes do. But whatever the rule on that, there's no trace of XRPermissionDescriptor in Chromium, Gecko or WebKit so this removal is fine.

Reply:
>This is a dictionary, which I don't think we should have in BCD but sometimes do.

Is there a guideline for the "sometimes do" part?

Reply:
My opinion is that they should never be in BCD, like mixins, but there's no guideline AFAIK. @ddbeck and @Elchi3 might have opinions too.

Reply:
Splitting out this and the following XR thing would unblock this PR I think.